apps/antalmanac/src/components/RightPane/CoursePane/CoursePaneButtonRow.tsx

```diff
@@ -109,19 +97,16 @@ export function ColumnToggleButton() {
             <FormControl>
                 <Select
                     multiple
-                    value={activeColumns}
+                    value={selectedColumnNames}
                     open={open}
                     onChange={handleChange}
                     onClose={handleClose}
                     renderValue={renderEmptySelectValue}
                     sx={{ visibility: 'hidden', position: 'absolute' }}
                 >
-                    {Object.entries(columnLabels).map(([column, label]) => (
+                    {columnLabelEntries.map(([column, label], idx) => (
```

Comment:
I like either `index` or `i`

```suggestion
                    {columnLabelEntries.map(([column, label], index) => (
```